osgar/record.py

```diff
@@ -81,16 +83,38 @@ def main():
         format='%(asctime)s %(name)-12s %(levelname)-8s %(message)s',
         datefmt='%Y-%m-%d %H:%M',
     )
-    parser = argparse.ArgumentParser(description='Record run on real HW with given configuration')
-    parser.add_argument('config', nargs='+', help='configuration file')
+    parser = argparse.ArgumentParser(description='Record run given configuration', add_help=False)
```

Comment:
why are you overriding default "help"?

Reply:
I see - you want to allow "unknown arguments" ... OK

Reply:
Because when I don't, I am not able to load the config file and provide help also for the arguments defined in the config file.